docs/Reference/Server.md

```diff
@@ -1640,6 +1641,45 @@ const fastify = require('fastify')({
 The handler is bound to the Fastify instance and is fully encapsulated, so
 different plugins can set different logger factories.
 
+#### setGenReqId
+<a id="set-gen-req-id"></a>
+
+`fastify.setGenReqId(function (rawReq))` Function for setting the request-id for
+additional Fastify instances. It will receive the _raw_ incoming request as a
+parameter. This function is expected to be error-free.
```

Comment:
all functions should be "error-free". Do you mean, that in any case should not throw an error?
"The provided function should not throw an Error in any case." ?